scripts/cassandra-cpp-driver.control

```diff
@@ -0,0 +1,7 @@
+Package: cassandra-cpp-driver
+Version: VERSION
+Section: base
+Priority: optional
+Architecture: amd64
+Maintainer: Shotover team
```

Comment:
What is "Shotover" here?

Reply:
Oops, that got left in from the project I originally wrote it for, updated.